platform/javascript/express/opentelemetry/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "express",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "@opentelemetry/api": "^1.0.3",
+    "@opentelemetry/auto-instrumentations-node": "^0.25.0",
+    "@opentelemetry/exporter-zipkin": "^1.0.0",
+    "@opentelemetry/sdk-node": "^0.26.0",
```

Comment:
Could we move this to the Dockerfile as in the instrumented app? It would improve build times by reusing and sharing the cache for all other dependencies with the baseline and Sentry-instrumented apps.

https://github.com/getsentry/sentry-sdk-benchmark/blob/5346d1a8a49676ca5f4376f36a732f26d3213a65/platform/javascript/express/instrumented/express-postgres.dockerfile#L7-L10